__tests__/integrations.test.js

```diff
@@ -347,6 +349,26 @@ describe('Articles', () => {
         });
     });
   });
-  
-  
 });
+
+// USERS
+describe('Users', () => {
+  describe('USERS: /api/users', () => {
+    test('GET 200: Endpoint returns all users', () => {
+      return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then(({ body: { users } }) => {
+          const expectedObject = {
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          };
```

Comment:
We do need a length check on the response array before going into the forEach as if the response was an empty array the forEach would never run and the expect statements would never run giving us a false positive.